validphys2/src/validphys/config.py

```diff
@@ -1628,13 +1671,12 @@ def produce_group_dataset_inputs_by_experiment(self, data_input):
     def produce_group_dataset_inputs_by_process(self, data_input):
         return self.produce_group_dataset_inputs_by_metadata(data_input, "nnpdf31_process")
 
-    def produce_scale_variation_theories(self, theoryid, point_prescription):
+    def produce_scale_variation_theories(self, point_prescription, t0id):
         """Produces a list of theoryids given a theoryid at central scales and a point
-        prescription. The options for the latter are '3 point', '5 point', '5bar point', '7 point'
-        and '9 point'. Note that these are defined in arXiv:1906.10698. This hard codes the
-        theories needed for each prescription to avoid user error."""
+        prescription. The options for the latter are defined in pointprescriptions.yaml.
+        This hard codes the theories needed for each prescription to avoid user error."""
```

Comment:
Actually I didn't pick this up before, why the scale variation use the t0 id and not the normal id?

Reply:
Both can work since the theoryids used for the scale variations depend on what is in pointprescriptions.yaml and scalevariationstheoryids.yaml. In scalevariationstheoryids the same lists of variations are repeated many times for a different `theoryid`. But whether this `theoryid` is the t0id or the regular theoryid is an arbitrary choice that's up to us (and I think both options are fine), but the benefit of using the t0id, is that we don't have to copy the list for all theoryids correspdonding to the alphas variations

Reply:
Could you add a comment in the docstr with "note that this is using the t0theoryid, which by default is equal to theoryid but can be set separately" or something like this.